lib/saml_idp/version.rb

```diff
@@ -1,4 +1,4 @@
 # encoding: utf-8
 module SamlIdp
-  VERSION = '0.4.0'
```

Comment:
We were previously "upgrading" the gem in IDP by pointing at a different SHA in git. However the "version" in the Gemfile.lock was reporting itself as `0.4.0` when in fact our tag was named `0.4.2` so I figured I would try to true-up the version the gem reports itself as